mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java

```diff
@@ -265,7 +264,7 @@ public static void validateData(List<Record> expected, List<Record> actual, int
     sortedExpected.sort(Comparator.comparingLong(record -> (Long) record.get(sortBy)));
     sortedActual.sort(Comparator.comparingLong(record -> (Long) record.get(sortBy)));
 
-    Assert.assertEquals(sortedExpected.size(), sortedActual.size());
+    Assertions.assertThat(sortedActual.size()).isEqualTo(sortedExpected.size());
```

Comment:
```suggestion
    assertThat(sortedActual).hasSameSizeAs(sortedExpected);
```

Reply:
the reason for this is that the content of both collections will be printed when the assertion ever fails, which makes debugging much easier. Please also update all the other affected places

Reply:
Yes. did it for all places where size was compared